### ISSUE TITLE
The setting of `//@grant` will cause `Object.defineProperties` error

### ISSUE THREAD

**Issue by F9y4ng** [Tampermonkey/tampermonkey#1428]
## Expected Behavior

Backwards compatibility with TM 4.13-

## Actual Behavior

If `// @grant none` is not set, Object.defineProperties(MouseEvent.prototype, {}) does not take effect.

## Specifications

- Chrome: 98.0.4758.102
- TM:  4.14
- OS:  WIN10 / x64

## Script

**The following code does not execute properly**

```js
// ==UserScript==
// @name         New Userscript
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  try to take over the world!
// @author       You
// @include      *
// @icon         data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==
// @grant        GM_info
// @run-at       document-start
// ==/UserScript==

(function () {
  "use strict";
  setTimeout(() => {
    Object.defineProperties(MouseEvent.prototype, {
      clientX: {
        get: function () {
          return this.x / 2;
        },
        configurable: false,
      },
      clientY: {
        get: function () {
          return this.y / 2;
        },
        configurable: false,
      },
    });
    document.addEventListener("mouseover", e => {
      console.log(e.x, e.clientX);
    });
  }, 100);
})();
```

**But, the following code can be executed normally**
```js
// ==UserScript==
// @name         New Userscript
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  try to take over the world!
// @author       You
// @include      *
// @icon         data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==
// @grant        none
// @run-at       document-start
// ==/UserScript==

(function () {
  "use strict";
  setTimeout(() => {
    Object.defineProperties(MouseEvent.prototype, {
      clientX: {
        get: function () {
          return this.x / 2;
        },
        configurable: false,
      },
      clientY: {
        get: function () {
          return this.y / 2;
        },
        configurable: false,
      },
    });
    document.addEventListener("mouseover", e => {
      console.log(e.x, e.clientX);
    });
  }, 100);
})();
```

**Comment by derjanb** [comment]
Should be fixed at 4.15.6153 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_4_15_6153.crx))

**Comment by F9y4ng** [comment]
> Should be fixed at 4.15.6153 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_4_15_6153.crx))

I tested 4.15.6153, and it was indeed fixed, thanks.
